### ISSUE TITLE
A11y_SQLAzureDataStudio_AddAzureAccount_AddAzureAccount_Keyboard:The tree view is announced as table

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9234]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#MacOs;#MAS1.3.1;#VoiceOver;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
Mac OS:10.15.3
Screen reader: Voice over.
MAS References:MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Connections from left menu.
4. Select Azure, navigate to + button & hit enter.
5. Navigate to Add account button in right side window & hit enter.
6. Launch screen reader as Voice over 

 
**Actual:**
On navigating to the tree view items "Steffi " it is announced as table.
Each item is announced as list box.
There is not level announced for the items.

**Expected:**
On navigating to the tree view items "Steffi " it is not a table so the word table should not be announced.
Each item should be announced as a button
There should be proper hierarchy levels for the items.

**User Impact:**
Screen reader users will receive the wrong information of the structure as well as the control and hierarchy will be conveyed in a wrong manner.

**Attachment link for Reference :**
![image](https://user-images.githubusercontent.com/24580570/74931784-5cb31a80-5406-11ea-848f-924e46ffdeda.png)
![image](https://user-images.githubusercontent.com/24580570/74931818-6fc5ea80-5406-11ea-957f-d07ca1db08d7.png)
![image](https://user-images.githubusercontent.com/24580570/74931849-82402400-5406-11ea-9958-700f13dc2fa2.png)

**Comment by alanrenmsft** [comment]
this is a voice over issue, this is a tree view and we have the role and level set properly for the tree.

**Comment by BhanuPrakashGit** [comment]
@alanrenmsft Could you please reopened the bug as it still repro.
As this is External to Voice over. Please share the sample app so that we can route to the external team.

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-MAC-Feb2020;

**Comment by fsteffi** [comment]
#closed;
issue still repro logged new bug https://github.com/microsoft/azuredatastudio/issues/22493
